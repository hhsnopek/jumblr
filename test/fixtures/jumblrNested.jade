include ../../lib/jumblr

+jumblr('Video')
  li.video
    +jumblr('Caption')
      a.caption
