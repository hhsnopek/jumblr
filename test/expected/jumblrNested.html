{block:Video}<li class="video">{block:Caption}<a class="caption"></a>{/block:Caption}</li>{/block:Video}
